packages/client/lib/rpc/modules/eth.ts

```diff
@@ -39,24 +72,189 @@ export class Eth {
       [[validators.hex, validators.blockHash]]
     )
 
+    this.getCode = middleware(this.getCode.bind(this), 2, [
+      [validators.address],
+      [validators.blockOption],
+    ])
+
+    this.getStorageAt = middleware(this.getStorageAt.bind(this), 3, [
+      [validators.address],
+      [validators.hex],
+      [validators.blockOption],
+    ])
+
+    this.getTransactionCount = middleware(this.getTransactionCount.bind(this), 2, [
+      [validators.address],
+      [validators.blockOption],
+    ])
+
     this.protocolVersion = middleware(this.protocolVersion.bind(this), 0, [])
   }
 
   /**
    * Returns number of the most recent block.
-   * @param  {Array<*>} [params] An empty array
-   * @return {Promise}
+   * @param params An empty array
    */
   async blockNumber(_params = []) {
     const latestHeader = await this._chain.getLatestHeader()
     return `0x${latestHeader.number.toString(16)}`
   }
 
   /**
-   * Returns information about a block by block number
-   * @param  {Array<BN|bool>} [params] An array of two parameters: An big integer of a block number and a
-   * boolean determining whether it returns full transaction objects or just the transaction hashes
-   * @return {Promise}
+   * Executes a new message call immediately without creating a transaction on the block chain.
+   * Currently only "latest" block is supported.
+   * @param params An array of two parameters:
+   *   1. The transaction object
+   *       * from (optional) - The address the transaction is sent from
+   *       * to - The address the transaction is directed to
+   *       * gas (optional) - Integer of the gas provided for the transaction execution
+   *       * gasPrice (optional) - Integer of the gasPrice used for each paid gas
+   *       * value (optional) - Integer of the value sent with this transaction
+   *       * data (optional) - Hash of the method signature and encoded parameters.
+   *   2. integer block number, or the string "latest", "earliest" or "pending"
+   * @returns The return value of the executed contract.
+   */
+  async call(params: [RpcTx, string]) {
+    const [transaction, blockOpt] = params
+
+    if (!this._vm) {
+      throw new Error('missing vm')
+    }
```

Comment:
Some nit: this error message - if thrown - will likely lead to confusion by the user and should be replaced on occasion with something more descriptive like "client instance is running in a sync mode where this RPC call is not supported" (attention! first shot! 😄  ) or something.

No blocker though.